src/app/page.tsx

```diff
@@ -7,12 +7,16 @@ import { ArrowLongLeftIcon, FunnelIcon } from "@heroicons/react/16/solid";
 import PincsHeader from "@/components/PincsHeader";
 import PincsButton from "@/components/PincsButton";
 import Link from "next/link";
+import { useSession } from "next-auth/react";
 
 export default function Home() {
+  const session = useSession();
   const [filterDrawerOpen, setFilterDrawerOpen] = useState<boolean>(false);
   const [selectedSubjects, setSelectedSubjects] = useState<Tag[]>([]);
   const subjects = Object.values(Subject);
 
+  console.log("session: ", session);
```

Comment:
Happy to remove this; figured it might be helpful until we know auth works locally and on the server.